e2etests/pkg/config/from_containers.go

```diff
@@ -0,0 +1,32 @@
+// SPDX-License-Identifier:Apache-2.0
+
+package config
+
+import (
+	frrk8sv1beta1 "github.com/metallb/frrk8s/api/v1beta1"
+	"go.universe.tf/e2etest/pkg/frr"
+	frrcontainer "go.universe.tf/e2etest/pkg/frr/container"
+	"go.universe.tf/e2etest/pkg/ipfamily"
+)
+
+func NeighborsForContainers(frrs []*frrcontainer.FRR, modify ...func(frr.Neighbor)) []frrk8sv1beta1.Neighbor {
+	res := make([]frrk8sv1beta1.Neighbor, 0)
+	for _, f := range frrs {
+		addresses := f.AddressesForFamily(ipfamily.IPv4)
+		ebgpMultihop := false
+		if f.NeighborConfig.MultiHop && f.NeighborConfig.ASN != f.RouterConfig.ASN {
+			ebgpMultihop = true
+		}
+
+		for _, address := range addresses {
+			neigh := frrk8sv1beta1.Neighbor{
+				ASN:          f.RouterConfig.ASN,
+				Address:      address,
+				Port:         f.RouterConfig.BGPPort,
+				EBGPMultiHop: ebgpMultihop,
+			}
+			res = append(res, neigh)
```

Comment:
Question: I know we're only testing ipv4 here, but in the case of dual-stack, we'll have 2 neighbors for the same frrcontainer. is that valid?

Reply:
because when we deploy dual stack, the primary ip is ipv4.
So, whatever works on ipv4 should work also on dual, because the ipv4 tests rely on the primary ip of the cluster being ipv4. That's the reason why we don't run ipv6 tests. 
The extra "dual" tests are supposed to test when we cover the secondary ip in some way.